rtrlib/transport/ssh/ssh_transport.c

```diff
@@ -243,11 +243,10 @@ int tr_ssh_recv_async(const struct tr_ssh_socket *tr_ssh_sock, void *buf, const
 	return rtval;
 }
 
```

Comment:
is commit log consistent with modified code ? (ssh_send and tr_ssh recv ?)

Reply:
fixed in my local branch